app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -21,8 +21,7 @@ public function transform(Post $post)
             'id' => $post->id,
             'status' => $post->status,
             'caption' => $post->caption,
-            // Add link to review reportback item in Rogue here once that page exists
-            // 'uri' => 'link_goes_here'
+            'uri' => url(config('services.phoenix.uri') . '/api/v1/reportbacks/'.$post->id, ['absolute' => true]),
```

Comment:
hmm yea, it I thought this was meant to be like the RBs page in phoenix, which we don't have a parallel for yet in rogue.